mlir/utils/tuna/tuna-script.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+# For installing mysql 8.0 for testing, or for running with an isolated database.
+function mysql_setup_generic
+{
+    # Latest "generic" version.  (Must have libaio1 installed already.)
+    # Note that all this happens without privileges.
+    pushd /tmp
+    wget https://dev.mysql.com/get/Downloads/MySQL-8.0/mysql-8.0.34-linux-glibc2.28-x86_64.tar.gz
+    tar xf mysql-8.0.34-linux-glibc2.28-x86_64.tar.gz
+    mv mysql-8.0.34-linux-glibc2.28-x86_64 mysql
+    export PATH=$PATH:/tmp/mysql/bin
+    mysqld --initialize-insecure --datadir=/tmp/mysql/data
+    # --pid-file=/tmp/mysql/mysqld.pid --socket=/tmp/mysql/mysqld.sock
+    mysqld -D --basedir=/tmp/mysql --datadir=/tmp/mysql/data --log-error=/tmp/mysql/errors.log
+    popd
+
+    # Using this name should force socket access, which we want.
+    TUNA_DB_HOSTNAME=localhost
+    mysql --user root -e 'create database tuna;'
+}
+
+function tuna_setup
+{
+    startdir=`pwd`
+    rm -rf MITuna
+    git clone --branch pf-tuna-rocmlir-3 https://github.com/ROCmSoftwarePlatform/MITuna.git
```

Comment:
Clone Tuna and do python things.  My development branch right now, rather than wait for the Tuna pull request.  A lot of python modules are installed, but at least it only happens once and they stick around for subsequent tuna-script.sh invocations in the same docker container.  Add our database tables at the end.

Reply:
Can we move this into the Jenkinsfile so that it picks up on any weird config that Jenkins does?

Reply:
I hesitate, because then any desired Tuna update will require a docker image re-generation.